ports/mimxrt/Makefile

```diff
@@ -49,12 +59,14 @@ CFLAGS += -DXIP_EXTERNAL_FLASH=1 \
 	-D__START=main \
 	-DCPU_HEADER_H='<$(MCU_SERIES).h>'
 
-LDFLAGS = $(addprefix -T,$(LD_FILES)) -Map=$@.map --cref
+LDFLAGS = $(addprefix -T,$(LD_FILES)) -Map=$@.map --cref --print-memory-usage
 LIBS = $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
 
 # Tune for Debugging or Optimization
 ifeq ($(DEBUG),1)
 CFLAGS += -O0 -ggdb
+LDFLAGS += --gc-sections
+CFLAGS += -fdata-sections -ffunction-sections
```

Comment:
why does this need to be added here?

Reply:
without it I could not see the symbols properly when debugging with JLink. It is "just" the DEBUG build so it has no influence on the release build